stage1/init/container.go

```diff
@@ -79,12 +80,30 @@ func LoadContainer(root string) (*Container, error) {
 	return c, nil
 }
 
+// quoteExec returns an array of quoted strings appropriate for systemd execStart usage
+func quoteExec(exec []string) string {
+	var qexec []string
+
+	if len(exec) > 0 {
+		//FIXME(vc): systemd gets angry if qexec[0] is quoted, which must be a bug.
+		qexec = append(qexec, exec[0])
+
+		if len(exec) > 1 {
+			for _, arg := range exec[1:] {
+				qexec = append(qexec, strconv.Quote(arg))
```

Comment:
I can't see a good way of doing this without rewriting a slimmed down strconv.Quote :-/